<launch>
    <node pkg="trajectory_generator" type="trajectory_generator_node" name="trajectory_generator_node"
          output="screen" >

        <remap from="~waypoints" to="/waypoint_generator/waypoints"/>

        <param name="planning/max_vel" value="4.0"/>
        <param name="planning/max_acc" value="3.0"/>
        <param name="planning/dev_order" value="3"/>
        <param name="planning/fixed_time" value="false"/>
        <param name="vis/vis_traj_width" value="0.10"/>

    </node>

    <node pkg="waypoint_generator"  type="waypoint_generator" name="waypoint_generator" output="screen">
        <remap from="~goal" to="/goal"/>
        <param name="waypoint_type" value="manual-lonely-waypoint"/>    
    </node>

    <node name="rviz" pkg="rviz" type="rviz" output="screen"
          args="-d $(find trajectory_generator)/launch/rviz_config/test_traj.rviz"/>
</launch>
